peerme/euroix_json.py

```diff
@@ -61,15 +61,13 @@ def fetch_json(self, ixp_json_file, timeout=10):
             ) for url in ixp_data_urls
         ]
         completed_tasks, _ = self.loop.run_until_complete(
-            asyncio.wait(http_tasks, timeout=10)
+            asyncio.wait(http_tasks, timeout=30)
```

Comment:
There isn't any passed in value... having a cli option is possible but kinda overkill, might add it as a config option (would rather not bloat the cli options).